### PR TITLE
prepare for data-anomalies page

### PR DESCRIPTION
**Pull request by sgratzl**
similar to https://github.com/cmu-delphi/www-main/pull/305, prepare for the new COVIDcast Data Anomalies page

### CODE REVIEW

File: content/covidcast/data-anomalies.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: COVIDcast Export Data
```
Comment (chinandrew) [inline content/covidcast/data-anomalies.md:2]: This is still the export data page info

Comment (sgratzl) [reply content/covidcast/data-anomalies.md]: my mistake, copy pasted a file and edited the original one

File: content/covidcast/export.md [hunk 2]
```diff
@@ -5,7 +5,6 @@ description: Use COVIDcast data in your own analysis
 layout: covidcast_app
 app_mode: export
 order: 6
-icon: solid/download
```
Comment (chinandrew) [inline content/covidcast/export.md:8]: what does this change do?

Comment (sgratzl) [reply content/covidcast/export.md:8]: it shouldn't have any effect, the line is a leftover an old version where the icons were still used

**Review by chinandrew — APPROVED** [review]
:shipit::shipit::shipit:

**Pull request merged**